TravelAgency-spring-mvc/src/main/webapp/WEB-INF/jsp/customer/setting.jsp

```diff
@@ -0,0 +1,61 @@
+<%-- 
+    Document   : edit
+    Created on : 17.12.2018, 10:54:02
+    Author     : Rajivv
+--%>
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="false" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="f" uri="http://java.sun.com/jsp/jstl/fmt" %>
+
+<my:pageTemplate title="Setting Account ${customer.name}">
+    <jsp:attribute name="body">
+        <form:form method="POST" action="/travel.agency/customer/settingAccount/${customer.id}" modelAttribute="customer">
```

Comment:
you are sending over settingCustomer not customer !! Change here or in the controller